app/views/content_items/_print_meta_data.html.erb

```diff
@@ -0,0 +1,33 @@
+<br/>
```

Comment:
It was originally used in whitehall and I think it was interacting with some other CSS used in the printing version of HtmlAttachments. Which is why I resorted to spacing. 

I've removed them and used CSS here instead.